### ISSUE TITLE
Mobile youtube for latest iOS

### ISSUE THREAD

**Issue by adewinne** [eddiemachado/bones#796]
I'm not a WP developer, however, just a basic include of a youtube url using vanilla bones will not work on the latest iOS 9.0 with the latest version of WP, which is currently 4.3.1. I've tested the exact same posting on iOS 6.1 (I know, very old) and it works fine. When I switch to a default WP theme the video works fine in iOS 9.0. It's only with bones that it doesn't work. Any advice?

**Comment by joshuaiz** [comment]
How are you including the youtube url? In a post or in a template? Full embed code or just the link? Does the youtube embed work on desktop with Bones?

**Comment by adewinne** [comment]
I've tried a number of different things.

How I'm including the youtube url, post or template:
- in the post as a youtube url on it's own line on it's own, which I believe uses WP's oEmbed method
- using plugins to see if I can work around the problem. Tried youtube-widget-responsive, youtube-embed-plus.
- with a custom field with type 'text'. In the posting I just enter the youtube url in the text field
- I've tried the full iframe embed code directly from the youtube video 

All of these have resulted in the same using bones theme:
- Works fine on desktop
- Does not work on iOS latest version on my iPhone 6
- Works on older iOS on my tablet

When I switch the theme to 'Twenty Fifteen' theme all scenarios work fine. 

For an actual example, see http://vvuzz.com/index.php/2015/09/15/the-best-car-ive-ever-driven/ . The video is near at the bottom of the article.

**Comment by joshuaiz** [comment]
You are correct, the video on your site will not play on iOS9 but does play on desktop.

I just tried a quick test embedding a YouTube video on a blank page and then trying to open in mobile on one of my sites using Bones. It does work...see here:

http://promo.vizualrecords.com/youtube/

The video size blows out the page layout but it still will play on iOS 9 Using iPhone 6/iOS 9.02.

Thus I think this has something to do with your CSS or some sort of responsive video plugin. I did a quick z-index check as this often can be the culprit if you have a sticky header/menu but that didn't seem to help. 

Try embedding a video on a completely blank page and see if that works on iOS. If so, then there is something in your CSS specific to that page that is causing it to not play. If it still doesn't play, my guess is it is something else in your CSS mobile styling that is the problem.

All that said, this is a Bones install from about 1 month ago so I don't know if anything's changed with the theme since then.

**Comment by adewinne** [comment]
I found the problem! It was the Custom Contact Forms plugin. I've verified that turning on this module will break videos in the latest iOS on iPhone. This url http://dev-bonesyoutube.pantheon.io/ has a base WP install with a vanilla bones install, both latest version. The video worked up until the point at which I installed and activated the Custom Contact Forms plugin.
